keps/993-two-phase-admission/README.md

```diff
@@ -253,6 +254,27 @@ The controller implementing a particular check should:
 * After approving the workload, keep an eye on the check if it starts failing,
   fail the check and cause the workload to move back to the suspended state.
 
+### Preemption Admission Check Controller
+
+In this proposal, the time to evict the preemption candidates varies based on the preemptor state
+the scheduler will not issue the eviction during it's process instead it will set a `Pending` admission check
+that is manged by a new built-in admission check controller.
+
+The **Admission Check Controller** will:
+
+- Watch for a change in state of the workloads pending preemption.
+- Watch for workloads that are finishing execution and therefore releasing quota.
+- Watch for AdmissionCheck changes, since the preemption policy can change.
+
+The preemption controller uses the kueue cache, since it needs to check the state of workloads admitted to the ClusterQueues.
```

Comment:
What is the difference between the admission check controller and the preemption controller? Are those the different components? Or one component?

Reply:
`Admission Check Controller` is a controller that manages a specific type of `AdmissionCheck`.

`preemption controller` is short for  `Preemption Admission Check Controller`.

Reply:
I see. Let's modify this section name, "Preemption Admission Check Controller" to the proper one.

Reply:
The section is named "Preemption Admission Check Controller" L257 but sure we can change L263 as well.

Reply:
Probably, "Admission Check controller" is https://github.com/kubernetes-sigs/kueue/blob/main/pkg/controller/core/admissioncheck_controller.go, and "preemption controller" is https://github.com/kubernetes-sigs/kueue/blob/ce7e628a1f6bb1e375790edd6fc7df3f7ac68958/pkg/controller/admissionchecks/preemption/preemption.go, right?

I wanted to clarify which sections do mention which controllers. 

So `but sure we can change L263 as well.` sounds good to me.